src/bin/minidump_dump.rs

```diff
@@ -57,6 +57,11 @@ fn print_minidump_dump(path: &Path) {
                 breakpad_info.print(stdout).unwrap();
             }
             // TODO: MemoryInfoList
+            if let Ok(crashpad_info) = dump.get_stream::<MinidumpCrashpadInfo>() {
+                crashpad_info.print(stdout).unwrap();
+            } else {
+                write!(stdout, "MinidumpCrashpadInfo cannot print invalid data").unwrap();
```

Comment:
This should probably print nothing if there's no `MinidumpCrashpadInfo` stream in the dump? But for that we'd need to distinguish between "no stream present" and "stream present, but parsing failed"

Reply:
Good catch. Would you believe me that I was thinking exactly the same while AFK? 😄